### PR TITLE
Project tech tags

### LINKED ISSUE #105: Add tech bubble tagging to projects
Each project should allow for tech bubble tagging

### PR DESCRIPTION
**Pull request by Istyatur**
Merge conflicts have been fixed
fixes #105

### CODE REVIEW

File: client/app/projects/profile/profile.controller.js [hunk 1]
```diff
@@ -200,6 +200,29 @@ angular.module('observatory3App')
             });
         }
     };
+//tech bubble code
+	$scope.isMentor = Auth.isMentor;
```
Comment (agundy) [inline client/app/projects/profile/profile.controller.js]: why do we only want mentors to add remove tech bubbles? Shouldn't anyone on the project be able to?

**Comment by Istyatur** [comment]
I don't know if you want anyone able to change it, which on project essentially is. I guessed safe, as who can do so is unspecified.

**Comment by Istyatur** [comment]
@agundy So do you want the unverified able to modify or not?

**Comment by Istyatur** [comment]
@agundy So... can you answer?

**Comment by agundy** [comment]
Sorry missed this. I'd say if you know how to do it feel free to add to this now with project members editing. If not leave it as just mentors and after merging this add another issue to resolve that

**Pull request closed without merging**